docs/make.jl

```diff
@@ -15,7 +31,13 @@ makedocs(;
     pages=[
         "Home" => "index.md",
         "User Guide" => "userguide.md",
-        "Examples" => "example.md",
+        "Examples" => [
+            "Kernel Ridge Regression" => "examples/kernel_ridge_regression.md",
+            "Training kernel parameters" => "examples/train_kernel_parameters.md",
+            "Gaussian process priors" => "examples/gaussianprocesspriors.md",
+            "SVM" => "examples/svm.md",
+            "Deep Kernel Learning" => "examples/deepkernellearning.md",
+        ],
```

Comment:
I think I would prefer if as in AbstractGPs the examples are listed after the regular documentation such that kernels, transforms, etc. are explained properly first. In addition, again the setup in AbstractGPs seems a bit better. I can't comment below, so I put the suggestion here but they should be moved below "API" in a second step:
```suggestion
        "Examples" => joinpath.("examples", filter(x -> endswith(x, ".md"), readdir(OUTPUT))),
```